packages/config/README.md

```diff
@@ -2,6 +2,48 @@
 
 ## Usage
 
+### `.bemrc` fields
+
+Field   | Type    | Purpose
+---     | ---     | ---
+root    | `Boolean` | Used to determine the root directory from the current one
+naming  | `string`, `Object` | Name of existing in [`naming.preset`](https://github.com/bem/bem-sdk/tree/master/packages/naming.presets) preset or custom definition
+levels  | `Array<LevelConf>` | List of known levels in the right order<br> (usually local) with their configurations
+sets    | `Object<string, SetConf>` | Named sets of layers to use in projects
```

Comment:
Levels/layers again :) What is layer? Why do we declare levels not layers above?

Reply:
"to use in projects" > "to build" is better I think

Reply:
Layer is a property of a level. So we defining all levels but use just some.

Reply:
Good example with images at _methodology_ will help I think